mne/digitization/_utils.py

```diff
@@ -0,0 +1,393 @@
+            raise ValueError(
+                'Points have coord_frame entries that are incompatible with '
+                'coord_frame=%i: %s.' % (coord_frame, str(tuple(bad_frames))))
+
+    with start_file(fname) as fid:
+        write_dig_points(fid, pts, block=True, coord_frame=coord_frame)
+        end_file(fid)
+
+
+def _read_dig_points(fname, comments='%', unit='auto'):
+    """Read digitizer data from a file.
+
+    If fname ends in .hsp or .esp, the function assumes digitizer files in [m],
+    otherwise it assumes space-delimited text files in [mm].
+
+    Parameters
+    ----------
+    fname : str
+        The filepath of space delimited file with points, or a .mat file
+        (Polhemus FastTrak format).
+    comments : str
+        The character used to indicate the start of a comment;
+        Default: '%'.
+    unit : 'auto' | 'm' | 'cm' | 'mm'
+        Unit of the digitizer files (hsp and elp). If not 'm', coordinates will
+        be rescaled to 'm'. Default is 'auto', which assumes 'm' for *.hsp and
+        *.elp files and 'mm' for *.txt files, corresponding to the known
+        Polhemus export formats.
+
+    Returns
+    -------
+    dig_points : np.ndarray, shape (n_points, 3)
+        Array of dig points in [m].
+    """
+    _check_option('unit', unit, ['auto', 'm', 'mm', 'cm'])
+
+    _, ext = op.splitext(fname)
+    if ext == '.elp' or ext == '.hsp':
+        with open(fname) as fid:
+            file_str = fid.read()
+        value_pattern = r"\-?\d+\.?\d*e?\-?\d*"
+        coord_pattern = r"({0})\s+({0})\s+({0})\s*$".format(value_pattern)
+        if ext == '.hsp':
+            coord_pattern = '^' + coord_pattern
+        points_str = [m.groups() for m in re.finditer(coord_pattern, file_str,
+                                                      re.MULTILINE)]
+        dig_points = np.array(points_str, dtype=float)
+    elif ext == '.mat':  # like FastScan II
+        from scipy.io import loadmat
+        dig_points = loadmat(fname)['Points'].T
+    else:
+        dig_points = np.loadtxt(fname, comments=comments, ndmin=2)
+        if unit == 'auto':
+            unit = 'mm'
+        if dig_points.shape[1] > 3:
+            warn('Found %d columns instead of 3, using first 3 for XYZ '
+                 'coordinates' % (dig_points.shape[1],))
+            dig_points = dig_points[:, :3]
+
+    if dig_points.shape[-1] != 3:
+        err = 'Data must be (n, 3) instead of %s' % (dig_points.shape,)
+        raise ValueError(err)
+
+    if unit == 'mm':
+        dig_points /= 1000.
+    elif unit == 'cm':
+        dig_points /= 100.
+
+    return dig_points
+
+
+def _write_dig_points(fname, dig_points):
+    """Write points to text file.
+
+    Parameters
+    ----------
+    fname : str
+        Path to the file to write. The kind of file to write is determined
+        based on the extension: '.txt' for tab separated text file.
+    dig_points : numpy.ndarray, shape (n_points, 3)
+        Points.
```

Comment:
I'm not sure of the exact name but you seem to call it `DigPoints` or `Digitalization points` after.